lib/routes/customerRoutes.js

```diff
@@ -99,10 +100,27 @@ function triggerSuspend (req, res, next) {
     .catch(next)
 }
 
+function updateTxCustomerPhoto (req, res, next) {
+  const customerId = req.params.id
+  const txId = req.params.txId
+  const tcPhotoData = req.body.tcPhotoData
+  const direction = req.body.direction
+
+  Promise.all([customers.getById(customerId), txs.getTx(txId, direction)])
+    .then(([customer, tx]) => {
+      if (!customer || !tx) { throw httpError('Not Found', 404) }
```

Comment:
Are we sure we want to throw a `404`? Doesn't seem right. Worst case it'd be better to throw something as a Bad Request - 400 or a custom code if we want the machine to handle this error in some way